_posts/2022-01-21-safe-releases.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Reducing thrash and increasing throughput by focusing on elegant releases
```

Comment:
Thoughts on "Elegant releases: Reducing thrash and increasing throughout" as a title? Gets the meat of the post up front.

Reply:
I dig it. I love developer happiness 😄